sdk/servicebus/service-bus/test/utils/testutils2.ts

```diff
@@ -514,7 +514,7 @@ export async function drainReceiveAndDeleteReceiver(receiver: Receiver<{}>): Pro
   }
 }
 
-function connectionString() {
+export function getConnectionString() {
```

Comment:
Name was bugging you? :)

Reply:
Ah, I can revert.
I was exporting this as part of another PR, which was split for this bug fix.

Reply:
Reverted